### PR TITLE
Correcting URL and other minor k8s versions

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Correcting URL and other minor k8s versions

cc: @kbhawkey

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -160,34 +160,34 @@ js = [
...
 
 [[params.versions]]
-fullversion = "v1.18.0"
+fullversion = "v1.18.8"
 version = "v1.18"
-githubbranch = "v1.18.0"
+githubbranch = "v1.18.8"
 docsbranch = "release-1.18"
-url = "https://v-18.docs.kubernetes.io"
+url = "https://v-1.18.docs.kubernetes.io"
```
Comment (justaugustus) [inline config.toml:171]:
```suggestion
url = "https://v1-18.docs.kubernetes.io"
```

?

Comment (kbhawkey) [inline config.toml:171]: nit: v1-18.docs.kubernetes.io ?

Comment (savitharaghunathan) [reply config.toml:171]: Thank you for catching it @justaugustus . This is like the 4th time this week I am masking mistakes on this particular url. Fix coming up right away

Comment (savitharaghunathan) [reply config.toml]: Hi @kbhawkey Thanks for your review. Fixed it :)

Comment (savitharaghunathan) [reply config.toml]: Fixed it

**Comment by jimangel** [comment]
Per discussion [here](https://kubernetes.slack.com/archives/C1J0BPD2M/p1598466973088100?thread_ts=1598447979.070600&cid=C1J0BPD2M), I am manually merging this PR as part of the 1.19 release. 🎉

**Pull request merged**